src/sagemaker/serve/mode/in_process_mode.py

```diff
@@ -69,21 +70,30 @@ def create_server(
         logger.info("Waiting for model server %s to start up...", self.model_server)
 
         if self.model_server == ModelServer.MMS:
-            self._ping_container = self._multi_model_server_deep_ping
+            self._ping_local_server = self._multi_model_server_deep_ping
+            self._start_serving()
 
-        time_limit = datetime.now() + timedelta(seconds=5)
-        while self._ping_container is not None:
-            final_pull = datetime.now() > time_limit
+        # allow some time for server to be ready.
+        time.sleep(1)
 
+        count = 1
+        time_limit = datetime.now() + timedelta(seconds=20)
```

Comment:
nit: `time_limit = datetime.now() + timedelta(seconds=5)`

Reply:
Will fix, thank you